scss/components/_header.scss

```diff
@@ -0,0 +1,55 @@
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  margin: 30px 0 40px 0;
+  width: 1400px;
+}
+
+.logo__image {
+  margin-top: 8px;
+  width: 188px;
+  height: 32px;
+}
+
+.nav__container {
+  display: flex;
+}
+
+.nav__block {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__link {
+  padding-top: 8px;
+  text-decoration: 0;
+  margin: 0 15px;
+  min-width: 66px;
+}
+
+.nav__btn-container {
+  display: flex;
+}
+
+.nav__container-button-in {
```

Comment:
be consistent with naming
.nav__btn-container
.nav__container-button-in
btn vs button

what does button-in mean? why _in_?